Automattic/jetpack
# Release Script: Check for remote release branch first

## Proposed changes:

Checks for an existing remote release branch first before proceeding with the script.

The release branch script itself checks if it exists locally and asks if you want to delete it already, so we just need to cover our bases for the remote branch. 
### Other information:


## Jetpack product discussion


n/a
## Does this pull request change what data or activity we track or use?


no
## Testing instructions:






Throw an exit around line 117 and run `tools/release-plugin -a plugins/jetpack 11.7`. It should see the existing release branch and ask you to delete it (I uh, wouldn't actually delete it though). Doing the same for 11.9 should let it continue.
